expo/expo
# [docs] Add missing library logos in API reference

# Why



Some of the libraries API references are missing logos. However, for these libraries, the logo is already provided in each of their README files.

# How



Add the missing logo for 
- Cellular
- Clipboard
- Contacts
- Device
- Document Picker
- Font
- Keep Awake
- Location
- Print
- Screen Orientation
- Sharing
- Speech
- Splash Screen
- SQLite
- Status Bar
- Store Review
- Updates

Changes backported to for all SDK versions currently available and unversioned.

**Example preview** after these changes

<img width="905" alt="CleanShot 2023-10-07 at 17 14 52@2x" src="https://github.com/expo/expo/assets/10234615/52092938-89c9-4f21-9ead-94827105377e">

<img width="1499" alt="CleanShot 2023-10-07 at 17 15 21@2x" src="https://github.com/expo/expo/assets/10234615/b74604f3-ee28-4f3a-a287-a521010c5d8d">

<img width="1512" alt="CleanShot 2023-10-07 at 17 15 31@2x" src="https://github.com/expo/expo/assets/10234615/b5bac0ce-ff37-4760-8179-38056487ade4">

<img width="1508" alt="CleanShot 2023-10-07 at 17 15 37@2x" src="https://github.com/expo/expo/assets/10234615/32915391-cb3d-41a3-9c43-8360194fc007">


# Test Plan



- Run docs locally
- Visit a library doc from the above list
- Check different SDK versions to verify that logo is displayed for that library 

# Checklist
